ProcessLib/TES/TESAssemblyParams.h

```diff
@@ -0,0 +1,83 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2016, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#ifndef PROCESS_LIB_TESPROCESS_NOTPL_H_
+#define PROCESS_LIB_TESPROCESS_NOTPL_H_
+
+
+#include <Eigen/Sparse>
+#include <Eigen/Eigen>
+
+#include "MaterialsLib/Adsorption/Reaction.h"
+
+#include "ProcessLib/VariableTransformation.h"
+
+
+namespace ProcessLib
+{
+
+namespace TES
+{
+
+const unsigned NODAL_DOF = 3;
+const unsigned COMPONENT_ID_PRESSURE      = 0;
+const unsigned COMPONENT_ID_TEMPERATURE   = 1;
+const unsigned COMPONENT_ID_MASS_FRACTION = 2;
+
+const double M_N2  = 0.028013;
+const double M_H2O = 0.018016;
+
+struct AssemblyParams
+{
+    Trafo trafo_p{1.0};
+    Trafo trafo_T{1.0};
+    Trafo trafo_x{1.0};
+
+    std::unique_ptr<Adsorption::Reaction> react_sys;
+
+    double fluid_specific_heat_source = std::numeric_limits<double>::quiet_NaN();
+    double cpG = std::numeric_limits<double>::quiet_NaN(); // specific isobaric fluid heat capacity
+
+    Eigen::MatrixXd solid_perm_tensor =
+            Eigen::MatrixXd::Constant(3, 3, std::numeric_limits<double>::quiet_NaN()); // TODO get dimensions
+    double solid_specific_heat_source = std::numeric_limits<double>::quiet_NaN();
+    double solid_heat_cond = std::numeric_limits<double>::quiet_NaN();
+    double cpS = std::numeric_limits<double>::quiet_NaN();    // specific isobaric solid heat capacity
+
+    double tortuosity = std::numeric_limits<double>::quiet_NaN();
+    double diffusion_coefficient_component = std::numeric_limits<double>::quiet_NaN(); // ???
+
+    double poro = std::numeric_limits<double>::quiet_NaN();
+
+    double rho_SR_dry = std::numeric_limits<double>::quiet_NaN();
+
+    const double M_inert = M_N2; // N2
+    const double M_react = M_H2O;
+
+    // TODO unify variable names
```

Comment:
Some are short, some are physical symbols, some are long. maybe they should be unified. But I'll do it in a subsequent PR.